src/_dev-execution/bootstrap.ts

```diff
@@ -75,5 +75,7 @@ import { TokenManager } from './token-manager';
         onConnectorAuthenticationRequested: (connectorId) => {
             return Promise.reject(new Error(`Authorization failed for ${connectorId}`));
         },
+        // userInterfaceID: '1b8eb88a-0c8d-4cec-a4d9-18a2d8359b89',
```

Comment:
do we need these comments?

Reply:
done c608a96ba67cf09e66b550b35e991a264f13bfac